synapse/handlers/identity.py

```diff
@@ -138,6 +140,52 @@ def bind_threepid(self, creds, mxid):
             data = json.loads(e.msg)
         defer.returnValue(data)
 
+    @defer.inlineCallbacks
+    def unbind_threepid(self, mxid, threepid):
+        """
+        Removes a binding from an identity server
+        Args:
+            mxid (str): Matrix user ID of binding to be removed
+            threepid (dict): Dict with medium & address of binding to be removed
+
+        Returns:
+            Deferred
```

Comment:
deferred what?

`Deferred[bool]` I guess, but what does the bool mean?

Reply:
fixed